frontend/amundsen_application/static/js/components/ResourceListItem/FeatureListItem/index.tsx

```diff
@@ -31,11 +30,9 @@ const FeatureListItem: React.FC<FeatureListItemProps> = ({
   feature,
   logging,
 }: FeatureListItemProps) => {
-  let source = '';
-  if (feature.availability) {
-    source = feature.availability.length > 0 ? feature.availability[0] : '';
-  }
 
+  const source =
+    feature.availability?.length > 0 ? feature.availability[0] : '';
```

Comment:
@allisonsuarez Fixed the same line just now by adding a `?`.

Reply:
we first have to check that availability is not null because it doesn't return [] it returns null if there's no availability
https://github.com/amundsen-io/amundsen/pull/1276

Reply:
oooh
I see

Reply:
I didn't realize you could do that in Ts, if you can make the same change in the 3 places where we have this that would be great

Reply:
This is really a bandaid on the frontend. `availability: string[]` should never be Null or Undefined according to the type definition and we shouldn't need to add a qualifying check. We just need to ensure the backend has the same definition.

Reply:
And yes, the `?.` operator was added in a recent version of TS, so it's new to me as well. Super useful.